app/Http/Controllers/Admin/AmaController.php

```diff
@@ -49,18 +50,40 @@ public function index(Request $request)
      */
     public function create()
     {
-        //
+        $ama_announcements = DB::table('ama_announcements')->select('title', 'user_id')->get();
+        return view('create', ['ama_announcements' => $ama_announcements]);
     }
 
     /**
-     * Store a newly created resource in storage.
-     *
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+
+    public function save(Request $request)
     {
-        //
+
+        $this->validate($request, [
+            'title'   => 'required',
+            'content' => 'required',
+        ]);
+
+        $amas = new Ama();
+
+        $amas->title   = $request->input('title');
+        $amas->content = $request->input('content');
+
+        if ($request->tags) {
+            $amas->tags = $request->input('tags');
+        }
+        else{
+            $amas->tags = "";
+        }
+
+        $amas->person              = Auth::user()->name;
```

Comment:
Person is the person who will be answering questions - you should set it from the frontend. Make it an input field (so the admin can enter Ceca Srbova as a person who will be answering).